nixos/doc/manual/default.nix

```
@@ -63,6 +63,9 @@ let
       optionIdPrefix = "test-opt-";
     };
 
+  testDriverMachineDocstrings = pkgs.callPackage
+    "${prefix}/nixos/lib/test-driver/nixos-test-driver-docstrings.nix" {};
```

OfBorg failed with

```
       … while realising the context of path '/nix/store/3jdj37ibg4nz1srpx6q9cl4siv2xam2a-ofborg-evaluator-0/nixos/lib/test-driver/nixos-test-driver-docstrings.nix'

       at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-0/lib/customisation.nix:124:45:

          123|     let
          124|       f = if lib.isFunction fn then fn else import fn;
             |                                             ^
          125|       fargs = lib.functionArgs f;

       error: path '/nix/store/3jdj37ibg4nz1srpx6q9cl4siv2xam2a-ofborg-evaluator-0' is not valid
```

I think it's not happy about importing from a copied path, and indeed we shouldn't cause extra copying into the store.

This seems to work fine:

```suggestion
    ../../nixos/lib/test-driver/nixos-test-driver-docstrings.nix {};
```